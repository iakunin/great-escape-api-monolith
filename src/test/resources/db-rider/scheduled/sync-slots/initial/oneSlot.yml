slot:
  -
    date_time_local: "2021-04-21 10:00:00.000000"
    date_time_with_time_zone: "2021-04-21 07:00:00.000000"
    is_available: true
    price: 123123
    quest_id: "343d2fdd-3f5f-4aed-9f23-092057720bd2"
