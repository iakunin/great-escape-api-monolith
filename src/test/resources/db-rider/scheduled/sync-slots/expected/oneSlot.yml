slot:
  -
    date_time_local: "2021-04-21 10:00:00.000000"
    is_available: false
    price: 3000
    quest_id: "343d2fdd-3f5f-4aed-9f23-092057720bd2"
